code/modules/mob/living/death.dm

```diff
@@ -19,6 +19,23 @@ GLOBAL_VAR_INIT(permadeath, FALSE)
 	SEND_SIGNAL(src, COMSIG_LIVING_GIBBED, no_brain, no_organs, no_bodyparts)
 	qdel(src)
 
+/mob/living/proc/gib_no_drop(no_brain, no_organs, no_bodyparts)
```

Comment:
Add an argument for this to `gib()` instead

Reply:
but I cant because of the proc
![image](https://github.com/user-attachments/assets/bb9255a5-edee-413c-8b22-8f8346ec3e87)

Reply:
nvm I see what you mean, done